ank/src/filtered_complete_state.rs

```diff
@@ -136,3 +137,5 @@ impl From<ank_base::Workload> for FilteredWorkloadSpec {
         }
     }
 }
+
+// TODO: add helper function for generating filtered complete state
```

Comment:
There are indeed multiple locations where such a function would be needed. Another possibility would be to add the `generate_test_proto_complete_state` fn and convert to filtered compete state in the tests (.into())